### PR TITLE
Enable 'approve' plugin for all sriov operator related repos

### PR DESCRIPTION
**Pull request by pliurh**

### CODE REVIEW

**Comment by pliurh** [comment]
@petr-muller Could help to take a look?

File: core-services/prow/02_config/_config.yaml [hunk 1]
```diff
@@ -847,6 +847,10 @@ tide:
     - openshift-priv/service-ca-operator
     - openshift-priv/service-catalog
     - openshift-priv/sig-storage-local-static-provisioner
+    - openshift-priv/sriov-cni
+    - openshift-priv/sriov-dp-admission-controller
+    - openshift-priv/sriov-network-device-plugin
+    - openshift-priv/sriov-network-operator
```
Comment (petr-muller) [inline core-services/prow/02_config/_config.yaml:853]: Don't move the `openshift-priv` ones - they will be moved automatically along with other necessary changes. This caused the config check failures - you would need to also take care of setting up plugins for `openshift-priv/*` repos. Just take care of everything `openshift/*` and the tools will do the rest.

Comment (pliurh) [reply core-services/prow/02_config/_config.yaml]: fixed.

**Comment by petr-muller** [comment]
The change is fine now, but the code changes are for some reason amended into a merge commit. Please fix that (so that your changes are in a separate commit) and we can merge this.

**Comment by pliurh** [comment]
> The change is fine now, but the code changes are for some reason amended into a merge commit. Please fix that (so that your changes are in a separate commit) and we can merge this.

Sorry, it's a mistake. Fixed.

**Pull request merged**